### DISCUSSION TITLE
How to download JUnit 5

### DISCUSSION THREAD

**Discussion by harmin-parra** [junit-team/junit-framework#3550] [Q&A] [answered]
Hi,

I am not an expert in Java, Maven Gradle, etc,
So, this question may sound ridiculous to some of you.

How can I download JUnit5 ?

Where can I find the JAR file ?

**Comment by sormuras** [comment, marked as answer]
Ho Harmin, 

have a look at https://junit.org/junit5/docs/current/user-guide/#running-tests-console-launcher which has links to the "raw uber" JAR file.

This answer https://stackoverflow.com/a/52373592/1431016 also has an example.

**Reply by harmin-parra** [reply]
Thank you

**Reply by saiema** [reply]
Hello, those links use JUnit 4.13.2, not JUnit 5, is there a similar link for JUnit 5?

**Reply by sormuras** [reply]
With

> JUnit 5 = JUnit Platform + JUnit Jupiter + JUnit Vintage

and

> An executable Fat JAR (`junit-platform-console-standalone-1.12.1.jar)` that contains the contents of all of its dependencies is published in the [Maven Central](https://central.sonatype.com/) repository under the [junit-platform-console-standalone](https://repo1.maven.org/maven2/org/junit/platform/junit-platform-console-standalone) directory. It contains the contents of the following artifacts:
> [...]

you get "JUnit 5" in an all-batteries-included package (including JUnit 4.13.2).

`$ java -jar junit-platform-console-standalone-1.12.1.jar execute <OPTIONS>`
```
├─ JUnit Vintage
│  └─ example.JUnit4Tests
│     └─ standardJUnit4Test ✔
└─ JUnit Jupiter
   ├─ StandardTests
   │  ├─ succeedingTest() ✔
   │  └─ skippedTest() ↷ for demonstration purposes
   └─ A special test case
      ├─ Custom test name containing spaces ✔
      ├─ ╯°□°)╯ ✔
      └─ 😱 ✔
```

**Reply by saiema** [reply]
Oh sorry, my bad.

Thank you very much!
